test/integration/smoke/test_network_ipv6.py

```diff
@@ -394,7 +394,7 @@ def getRouterProcessStatus(self, router, cmd):
             cmd,
             hypervisor=self.routerDetailsMap[router.id]['hypervisor']
         )
-        self.assertTrue(type(result) == list and len(result) > 0,
+        self.assertTrue(type(result) == list,
```

Comment:
this won't break tests?

Reply:
I changed ssh_command to use `ssh -q` in tools/marvin/marvin/lib/utils.py
due to it, some warning messages are not returned, for example
```
Warning: Permanently added '10.0.xx.xx' (ED25519) to the list of known hosts.
```

therefore it is ok if the returned result is empty
The only regression I found is with `test/integration/smoke/test_network_ipv6.py`, no other issues found (according to the smoke test results)